AD419.DataHelper.Web/Views/Titles/Delete.cshtml

```diff
@@ -104,7 +104,10 @@
         </dt>
 
         <dd>
-            @Html.DisplayFor(model => model.StaffType)
+            @{
+                var staffType = Model.StaffType;
+                @(string.IsNullOrWhiteSpace(staffType) ? Html.Raw("None") : Html.Raw(staffType));

```

Comment:
@taylorkj where does staff type come from and why does it have to be raw?  This seems dangerous.